components/centraldashboard/Dockerfile

```diff
@@ -1,18 +1,35 @@
-FROM node:10
+# Step 1: Builds and tests
+FROM node:10-alpine AS test
```

Comment:
What's the difference?

Reply:
Sorry, didn't see this comment! Alpine is a smaller image that the base node image (https://hub.docker.com/_/node/) so this reduces the size of our build quite a bit.

Reply:
Gotcha, thanks!